### PR TITLE
Release 8.0.3

### PR DESCRIPTION
**Pull request by root-expert**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,20 @@ All notable changes to this project will be documented in this file.
...
 These should not affect the functionality of the module.
 
-## [v8.0.2](https://github.com/voxpupuli/puppet-php/tree/v8.0.2) (2021-09-05)
+## [v8.0.3](https://github.com/voxpupuli/puppet-php/tree/v8.0.3) (2022-03-20)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-php/compare/v8.0.2...v8.0.3)
+
+**Fixed bugs:**
+
+- Allow a setting to be undef to force absent. [\#647](https://github.com/voxpupuli/puppet-php/pull/647) ([jadestorm](https://github.com/jadestorm))
```
Comment (smortex) [inline CHANGELOG.md:13]: This *sounds* like en enhancement, but is indeed a bugfix…  Maybe it's worth to rephrase the RP title to emphasis on the fix e.g. "Fix forcing a php::setting to be absent by setting its value to undef"?

But I can really live without it :-)

**Review by smortex — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (root-expert) [reply CHANGELOG.md:13]: Sounds good!

**Review by bastelfreak — APPROVED** [review]

**Comment by root-expert** [comment]
I would like to merge #648 before releasing if it is ready.
Also @bastelfreak, any ideas about the Arch failure?

**Comment by bastelfreak** [comment]
arch linux acceptance tests are quite new and not all tests are adjusted. please ignore those failures for now.

**Comment by bastelfreak** [comment]
@root-expert I merged the modulesync, can you please rebase?

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**